src/workerd/util/mimetype.c++

```diff
@@ -489,4 +434,18 @@ kj::Maybe<MimeType> MimeType::extract(kj::StringPtr input) {
   return kj::mv(mimeType);
 }
 
+kj::String MimeType::formDataWithBoundary(kj::StringPtr boundary) {
+  ToStringBuffer buffer(128);
+  // Note that the expectation is that the boundary is already properly formed
+  // and does not need any additional quoting or escaping.
```

Comment:
can we add a debug assertion here?

Reply:
not without additional unnecessary complexity. This is only called from one place.